OpenTabletDriverPlugins/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "OpenTabletDriverPlugins": {
+      "commandName": "Executable",
+      "executablePath": "C:\\Users\\fabio\\Fabio\\Programing\\OpenTabletDriver\\OpenTabletDriver\\bin\\Debug\\netcoreapp3.1\\OpenTabletDriver.exe",
+      "commandLineArgs": "-c C:\\Users\\fabio\\Fabio\\Programing\\OpenTabletDriver\\TabletDriverLib\\Configurations"
```

Comment:
Local path references, this needs to be fixed. I assume that this is just a VS20XX thing.